terraform/helm/fullnode/templates/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "aptos-fullnode.fullname" . }}-ingress
```

Comment:
I'd add `annotations` and labels for metadata as well, see example https://github.com/aptos-labs/aptos-core/blob/main/terraform/helm/fullnode/templates/service.yaml#L5-L10